src/main/java/edu/bu/metcs673/trackr/repo/TrackrUserRepository.java

```diff
@@ -13,7 +13,7 @@
  *
  */
 @Repository
-public interface UserRepository extends JpaRepository<User, Long> {
+public interface TrackrUserRepository extends JpaRepository<TrackrUser, Long> {
```

Comment:
Same comment about prefixing everything by Trackr.

Reply:
it was renamed to avoid a naming clash with spring-security "User" class.  the application User class was renamed and other classes related to that entity have been renamed to avoid confusion.